### ISSUE TITLE
0.9 is broken in ESM

### ISSUE THREAD

**Issue by Gozala** [ucan-wg/ts-ucan#71]
Here is what I'm getting

```
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/gozala/Projects/js-dag-ucan/node_modules/ucans/dist/esm/attenuation' imported from /Users/gozala/Projects/js-dag-ucan/node_modules/ucans/dist/esm/index.js
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:416:11)
    at moduleResolve (node:internal/modules/esm/resolve:932:10)
    at defaultResolve (node:internal/modules/esm/resolve:1044:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:422:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:222:40)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:76:40)
    at link (node:internal/modules/esm/module_job:75:36)
```

Problem is lack of `.js` extensions in imports see

https://unpkg.com/browse/ucans@0.9.0/dist/esm/index.js

**Comment by Gozala** [comment]
I end up working it around by downgrading to cjs https://github.com/ipld/js-dag-ucan/blob/main/test/ts-ucan.cjs

**Comment by Gozala** [comment]
Turning that into .js file and using import instead of require was not working for me

**Comment by icidasset** [comment]
@Gozala We've released `0.9.1` which should work as intended.

I've renamed your `test/ts-ucan.cjs` file to `ts-ucan.js` and changed the contents to the following:
```ts
import * as lib from "ucans"
export { lib }
export * from "ucans"
```
And all your tests from js-dag-ucan seem to pass.
